tee-worker/litentry/core/vc-task/receiver/src/lib.rs

```diff
@@ -529,7 +529,10 @@ where
 		let id_graph_hash = H256::from(blake2_256(&id_graph.encode()));
 		let assertion_networks = assertion.get_supported_web3networks();
 		let identities = get_eligible_identities(id_graph.as_ref(), assertion_networks);
-		ensure!(!identities.is_empty(), "No eligible identity".to_string());
+
+		if identities.is_empty() {
+			warn!("[RequestVc] : No eligible identities");
```

Comment:
same here.

Reply:
The previous logic returns an error if the response is empty, but there is already code to process the empty response in `tee-worker/litentry/core/credentials/src/brc20/amount_holder.rs#line124~131`, So this VC logic is wrong, should return an empty/false VC here.

![image](https://github.com/litentry/litentry-parachain/assets/120463031/503c6e6c-5163-4455-a9d9-987f8eb057c2)